docs/_includes/sidebar.md

```diff
@@ -18,6 +18,7 @@
       <li class="toctree-l2"><a href="{{ site.url | append: site.baseurl }}/guides/app-home">App Home</a></li>
       <li class="toctree-l2"><a href="{{ site.url | append: site.baseurl }}/guides/slash-commands">Slash Commands</a></li>
       <li class="toctree-l2"><a href="{{ site.url | append: site.baseurl }}/guides/events-api">Events API</a></li>
+      <li class="toctree-l2"><a href="{{ site.url | append: site.baseurl }}/guides/steps-from-apps">Workflow Steps</a></li>
```

Comment:
The Japanese version is not available yet. I will work on it when I have a chance.